src/transformers/trainer_pt_utils.py

```diff
@@ -675,6 +753,10 @@ def save_metrics(self, split, metrics, combined=True):
             The metrics returned from train/evaluate/predict
         combined (:obj:`bool`, `optional`, defaults to :obj:`True`):
             Creates combined metrics by updating ``all_results.json`` with metrics of this call
+
+    To understand the metrics please read the docstring of :obj:`transformers.Trainer.log_metrics()`. The only
```

Comment:
```suggestion
    To understand the metrics please read the docstring of :meth:`~transformers.Trainer.log_metrics`. The only
```
This will make a link the user can click.

Reply:
Actually :obj:`transformers.Trainer.log_metrics()` creates a clickable link too. 

But if the other approach is the one we use everywhere I will gladly switch to that. Thank you!

Reply:
Oh I didn't know obj tried to resolve the differences, that works for me then!

Reply:
Let's keep things consistent across the doc, so your suggestion seems to be the one then.

I'm still wanting to resolve that cross-referencing issue for inherited methods so that they are clickable, but didn't find enough time to sort it out. But that's when I learned about `the :obj:transformers.Trainer.log_metrics()`  method.